### PR TITLE
Added chrome driver

### PR DESCRIPTION
**Pull request by iLigonis**
Fixed issue where gecodriver won't close

### CODE REVIEW

File: YoCode/FrontEndCheck.cs [hunk 1]
```diff
@@ -5,60 +5,98 @@
...
 using System.IO;
+using Microsoft.Win32;
 
 namespace YoCode
 {
     internal class FrontEndCheck
     {
-        private readonly IWebDriver browser;
+        private static IWebDriver browser;
+        private static DriverService service;
```
Comment (mataszilaitis) [inline YoCode/FrontEndCheck.cs]: shouldn't this be local?

File: YoCode/FrontEndCheck.cs [hunk 2]
```diff
@@ -5,60 +5,98 @@
...
+        private static IWebDriver browser;
         private readonly string port;
+        private const string CHROME = "Google Chrome";
+        private const string FIREFOX = "Firefox";
+
+        public static bool running;
 
         public FrontEndCheck(string applicantsWebPort, string[] keyWords)
         {
+            if (string.IsNullOrEmpty(applicantsWebPort))
```
Comment (mikebroughton) [inline YoCode/FrontEndCheck.cs]: I think this method needs to be refactored. There are too many branching statements, which makes it hard to follow.

A good place to start would be to extract the body of each of the branches to a named function (starting with the long if statements in the middle).

Maybe:
```
TryLaunchFirefox()
TryLaunchChrome()
```
I wonder if rather than if else branches, it should try to launch Firefox and if that fails, try to launch chrome?

File: YoCode/FrontEndCheck.cs [hunk 3]
```diff
@@ -5,60 +5,98 @@
...
-                catch (Exception)
+                catch { };
+            }
+            else if (browsers.Any(a=>a.Contains(CHROME)))
+            {
+                try
                 {
-                    var chromeService = ChromeDriverService.CreateDefaultService(Directory.GetCurrentDirectory());
-                    chromeService.HideCommandPromptWindow = true;
+                    service = ChromeDriverService.CreateDefaultService(Directory.GetCurrentDirectory());
```
Comment (mikebroughton) [inline YoCode/FrontEndCheck.cs]: The first three lines of each of these branches are the same. That suggests they can be moved out of the if statement?

File: YoCode/FrontEndCheck.cs [hunk 4]
```diff
@@ -5,60 +5,98 @@
...
+            if (browser != null)
+            {
+                Console.WriteLine("browser closed");
                 browser.Dispose();
+                browser.Quit();
+                running = false;
+                return true;
             }
-            catch (WebDriverException e)
+            if(running)
```
Comment (mikebroughton) [inline YoCode/FrontEndCheck.cs]: The `running` variable is a boolean. When you can it's better to use boolean values directly. So rather than:

```
if(running) 
{
    return true;
} else
{
    return false;
}
```
You just need to:

```
return running;
```

**Comment by jogobom** [comment]
Closing because this PR has been replaced with code in #9.
